.gitignore

```diff
@@ -398,3 +398,5 @@ FodyWeavers.xsd
 
 # VS Code files
 .vscode/
+
+!/src/ChainSafe.Gaming/Web3/Core/Debug/
```

Comment:
Ah yes, now I remember. I had to explicity include that folder, because it includes some .cs files that are referenced troughout the project. Without those cs files project wouldn't compile.

Reply:
ok, got it